javascript/chronometer.js

```diff
@@ -1,34 +1,82 @@
 class Chronometer {
   constructor() {
     // ... your code goes here
+    this.currentTime =0
+    this.intervalId = null
   }
 
   start(callback) {
     // ... your code goes here
-  }
+    this.intervalId= setInterval(() => {
+      this.currentTime+=1;
+    
+     
+      if (this.current > 3) {
+        clearInterval(this.intervalId);
```

Comment:
Excelente Roman por ahi va esta funcion sin embargo hay que hacer la validacion y su gestion con la palabra callback que definimos en la funcion :eyes
```suggestion
if(callback) callback()
```